src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineControl_ModelComputation.cs

```diff
@@ -191,6 +193,8 @@ private async ValueTask DetermineHighlightedItemAndPresentItemsAsync(ImmutableSe
 
             if (symbolToSelect is not null)
                 symbolToSelect.IsSelected = true;
+            else
+                DocumentOutlineHelper.UnselectAll((IEnumerable<DocumentSymbolUIItem>)SymbolTree.ItemsSource);
```

Comment:
`SymbolTree.ItemsSource` can be null if `ComputeDataModelAsync` hasn't completed yet, we should only call this helper if `SymbolTree.ItemsSource` is not null